services/service-core/src/auth/keycloak.service.ts

```diff
@@ -29,6 +33,36 @@ export class KeycloakService {
     }
   }
 
+  async getAdminToken() {
+    const params = new URLSearchParams({
+      username: this.configService.get('KEYCLOAK_ADMIN'),
+      password: this.configService.get('KEYCLOAK_ADMIN_PASSWORD'),
+      grant_type: 'password',
+      client_id: 'admin-cli',
+      client_secret: this.configService.get('ADMIN_CLIENT_SECRET')
```

Comment:
What happens if tomorrow you decide to rename this variable in the config? Again, a magic string. The config object needs to be strongly typed or at least variable names should be in a separate enum.

better examples:
```
// best
this.configService.get().AdminClientSecret
```

```
// Also ok
this.configService.get(ConfigParam.ADMIN_CLIENT_SECRET)
```